contracts/src/access/ownable.rs

```diff
@@ -93,7 +93,10 @@ pub trait IOwnable {
     /// # Events
     ///
     /// Emits a [`OwnershipTransferred`] event.
-    fn transfer_ownership(&mut self, new_owner: Address) -> Result<(), Error>;
+    fn transfer_ownership(
+        &mut self,
+        new_owner: Address,
+    ) -> Result<(), Self::Error>;
```

Comment:
What is the need/benefit from doing `Self::Error` here vs just `Error`?

Reply:
We should use error type that is oriented around a trait.